deps/openssl/openssl/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+<!--
```

Comment:
Oy, I forgot to include this in the update instructions. The `openssl/.github' directory should not be copied over into deps. Please delete the entire `deps/openssl/openssl/.github` folder.

Reply:
I think I already did do that on the second commit because the `make` command in the openssl directory does not work otherwise.

Reply:
I have removed the .github and .travis and all other related dot files from the original commit. 
Please review and confirm if I have missed anything.